iOS/MSMapsModules/MSMapsModules/MSGeoJsonParser/MSGeoJsonParser.m

```diff
@@ -0,0 +1,138 @@
+//
+//  MSGeoJsonParser.m
+//  MapsModulesSample
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/12/20.
+//  Copyright © 2020 Microsoft.
+//  Licensed under the MIT license.
+//
+
+#import "MSGeoJsonParser.h"
+#import <MicrosoftMaps/MicrosoftMaps.h>
+
+@implementation MSGeoJsonParser {
+  MSMapElementLayer *layer;
+};
+
++ (MSMapElementLayer *)parse:(NSString *)geojson {
+  if (geojson == (id)[NSNull null] || geojson.length == 0) {
+    NSException *exception =
```

Comment:
No need for intermediate variable, just throw the exception on the same line it's created.